portfolio/src/main/webapp/index.html

```diff
@@ -6,28 +6,67 @@
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
   </head>
-  <body>
+  <body class="body">
+    <nav>
+       <div class="container">
+        <h1> Fred </h1>
+
+        <div class="menu">
+            <a href="index.html" class="is-active">Home</a>
+            <a href="about_me.html">About Me</a>
+           <!-- <a href="gallery.html">Gallery</a>-->
+            <a href="contact.html">Conatact</a>
+        </div>
+
+        <buttom class="ham">
```

Comment:
Any plan for this "ham" button?

Reply:
That is like for a hamburger menu that I was woking on, I haven't figured out how to make work just yet.